### ISSUE TITLE
Issue with setup

### ISSUE THREAD

**Issue by ed4wg** [rmusser01/MIS#1]
I've been trying to get this exact setup to work and ran across your article. Thanks for writing this up! The only thing i hadn't done was setup an IP address on the opt1 interface. Though, i don't think it's necessary. Anyway, i added it just to see if that was the issue with my setup, but it wasn't.

All that said, I'm only seeing broadcast traffic from LAN1 on the sniffer interface in security onion and not sure why. 

What is your host OS and what version of pfsense, vmware workstation are you running? I am on ubuntu 16.04, pfsense 2.4.2-RELEASE-P1 and the latest version of vmware workstation 14.1.1.

Do you have any suggestions? Maybe troubleshooting you did getting your setup to work? Thanks!

**Comment by rmusser01** [comment]
Hey, no problem. I had a similar issue, but I don't remember how I resolved it. (I did this as a lab at least 6months+ ago)

I can say that I was able to get things working with following setup:
Host OS: Fedora 24
VM: VMware Workstation12.5(I want to say, not sure.)
PFSense 2.3.4
Win10 Vanilla(nothing fancy/unpatched)
SecurityOnion.

I would suggest that you first start the pfsense vm, validate it can recieve traffic(this is where that external interface comes into play), then boot up security onion, again, validate all machines are getting traffic, and then finally, boot Win10.

That should result in pfsense being recognized by both, and sending traffic successfully from one subnet to another.
